_posts/2025-05-08-opensearch-performance-3.0.md

```diff
@@ -0,0 +1,1013 @@
+---
+layout: post
+title: "OpenSearch Project update: Performance progress in OpenSearch 3.0"
+layout: post
+authors:
+    - sisurab
+    - pallp
+    - vamshin
+    - seanzheng
+    - kolchfa
+date: 2025-05-08
+categories:
+        - technical-posts
+        - community
+meta_keywords: OpenSearch performance, benchmarking, query optimization, vector search, text queries, aggregations, OpenSearch roadmap
+meta_description: Explore the significant performance gains up through OpenSearch 3.0. Explore optimizations in text search, aggregations, range queries, and vector operations, and get a preview of the OpenSearch 2025 roadmap.
+has_science_table: true
+excerpt: Learn more about the strategic enhancements and performance features that OpenSearch has delivered up to version 3.0.
+featured_blog_post: false
+featured_image: false
+#additional_author_info: We sincerely appreciate the contributions to this blog from Anandhi Bumstead, Carl Meadows, Jon Handler, Dagney Braun, Michael Froh, Kunal Khatua, Andrew Ross, Harsha Vamsi, Bowen Lan, Rishabh Kumar Maurya, Sandesh Kumar, Marc Handalian, Rishabh Singh, Govind Kamat, Martin Gaievski, and Minal Shah.
+---
+
+OpenSearch 3.0 marks a major milestone in the project's ongoing performance journey and the first major release since 2.0 in April 2022. Building on the enhancements of the 2.x series, the 3.0 release integrates **Apache Lucene 10** and upgrades the Java runtime to **JDK 21**, bringing significant improvements to search throughput, indexing and query latency, and vector processing. With a **10x search performance boost** and **2.5x vector search performance boost**, Lucene 10 continues to be our strategic search library, and its latest version delivers measurable gains over earlier releases (baselined against 1.x) through enhanced query execution, skip-based filtering, and segment-level concurrency. 
+
```

Comment:
Link the words OpenSearch 3.0 to the release blog here: https://opensearch.org/blog/unveiling-opensearch-3-0/